packages/vm/src/wasm_backend/store.rs

```diff
@@ -15,29 +20,42 @@ use super::limiting_tunables::LimitingTunables;
 /// https://github.com/WebAssembly/memory64/blob/master/proposals/memory64/Overview.md
 const MAX_WASM_MEMORY: usize = 4 * 1024 * 1024 * 1024;
 
-/// Created a store with the default compiler and the given memory limit (in bytes)
+fn cost(operator: &Operator) -> u64 {
```

Comment:
Ah, we can set the costs for all operations. 
That is powerful. And a lot of responsibility to tune properly.

Please add an issue to review these numbers (even if you set them to something more realistic prior to merge, this is an important thing to review before 1.0)

Reply:
https://github.com/CosmWasm/cosmwasm/issues/670